src/DocumentFormat.OpenXml.Framework/Packaging/OpenXmlPart.cs

```diff
@@ -43,8 +43,23 @@ internal void Load(OpenXmlPackage? openXmlPackage, OpenXmlPart? parent, Uri uriT
 
             SetPackage(openXmlPackage, parent);
 
-            // TODO: should we delay load?
-            var part = _openXmlPackage.Features.GetRequired<IPackageFeature>().Package.GetPart(uriTarget);
+            IPackagePart? part = null;
+            try
+            {
+                // TODO: should we delay load?
+                part = _openXmlPackage.Features.GetRequired<IPackageFeature>().Package.GetPart(uriTarget);
```

Comment:
Could we make the Getpart return null when there's this failure? Then we can just check for a null